### PR TITLE
Add import support for self-describing backlog transfers

### PR DESCRIPTION
**Pull request by sevein**
This pull requests updates the `rebuild_transfer_backlog` management command to support restoring the new self-describing transfer packages. The biggest change is that it can be done over any pipeline even when it's freshly installed.

Directories are not populated yet - this work and other improvements may be completed in following pull requests.

Connects to https://github.com/archivematica/Issues/issues/446.
Connects to https://github.com/archivematica/Issues/issues/586.

### CODE REVIEW

File: src/dashboard/tests/test_rights.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+from metsrw.plugins import premisrw
+import pytest
+import uuid
+
+from components.rights import load
+from main.models import File, Transfer, SIP, MetadataAppliesToType
+
+
+_RIGHTS_STATEMENT_IDENTIFIER = (
```
Comment (cole) [inline src/dashboard/tests/test_rights.py]: This is all personal preference, but I don't know if we need to worry about "private" constants in test — I think it should be obvious to everybody not to import from a test file.

**Review by cole — APPROVED** [review]
This looks really good! Thanks for fixing all of the things 🙂 

What is the plan for the [TMP] commits, I'm assuming we don't want to merge them?

File: src/dashboard/tests/test_rights.py (hunk 1 above)
Comment (sevein) [reply src/dashboard/tests/test_rights.py]: You're right, fixed.

**Comment by sevein** [comment]
> What is the plan for the [TMP] commits, I'm assuming we don't want to merge them?
- The docker fix has been fixed by Radda, but it needs to be merged.
- I'm going to be uploading the instructions somewhere else and tell Peter and the client.
- pytest test collection improvement: I'll file a new issue for that!

**Pull request merged**